tests/standAloneClient/metadataObjectChanged.xml

```diff
@@ -0,0 +1,143 @@
+							<code>kConf::get("partner_notification_name")</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>creator_name</key>
+						<description>Entry creator name</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getkuser()->getFirstName() . ' ' . entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getkuser()->getLastName() : ''</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>creator_id</key>
+						<description>Entry creator ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getKuserId() : ''</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>creator_email</key>
+						<description>Entry creator email</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getkuser()->getEmail() : ''</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>owner_name</key>
+						<description>Account owner name</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getPartner()->getPartnerName() : ''</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>owner_email</key>
+						<description>Account owner email</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())->getPartner()->getAdminEmail() : ''</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>metadata_id</key>
+						<description>Metadata ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>$scope->getEvent()->getObject()->getId()</code>
+						</value>
+					</item>
+					<item objectType="KalturaEventNotificationParameter">
+						<key>entry_id</key>
+						<description>Entry ID</description>
+						<value objectType="KalturaEvalStringField">
+							<code>!is_null(entryPeer::retrieveByPk($scope->getEvent()->getObject()->getObjectId())) ? $scope->getEvent()->getObject()->getObjectId() : ''</code>
```

Comment:
No need to check here if is_null, you can always return the getObject()->getObjectId().